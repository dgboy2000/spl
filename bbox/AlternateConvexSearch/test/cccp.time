0.784936 6.078718
0.723425 12.086002
0.651774 18.810721
0.649483 25.004039
