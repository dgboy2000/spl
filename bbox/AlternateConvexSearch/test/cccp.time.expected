0.784936 4.880000
0.723425 9.700000
0.651774 15.090000
0.649483 20.040000
